m0t1fs/linux_kernel/st/m0t1fs_client_inc.sh

```diff
@@ -535,58 +535,58 @@ m0t1fs_crud()
 	local fsname2=$2
 	local fsname3=$3
 	local mode=$4
-	touch $MOTR_M0T1FS_MOUNT_DIR/$fsname1 || rc=1
-	touch $MOTR_M0T1FS_MOUNT_DIR/$fsname2 || rc=1
-	touch $MOTR_M0T1FS_MOUNT_DIR/$fsname3 || rc=1
-	stat  $MOTR_M0T1FS_MOUNT_DIR/$fsname1 || rc=1
-	stat  $MOTR_M0T1FS_MOUNT_DIR/$fsname2 || rc=1
-	stat  $MOTR_M0T1FS_MOUNT_DIR/$fsname3 || rc=1
-	chmod 567 $MOTR_M0T1FS_MOUNT_DIR/$fsname1 || rc=1
-	chmod 123 $MOTR_M0T1FS_MOUNT_DIR/$fsname2 || rc=1
-	chmod 345 $MOTR_M0T1FS_MOUNT_DIR/$fsname3 || rc=1
-	stat  $MOTR_M0T1FS_MOUNT_DIR/$fsname1 -c "%n: %a %s" || rc=1
-	stat  $MOTR_M0T1FS_MOUNT_DIR/$fsname2 -c "%n: %a %s" || rc=1
-	stat  $MOTR_M0T1FS_MOUNT_DIR/$fsname3 -c "%n: %a %s" || rc=1
-	dd if=/dev/zero of=$MOTR_M0T1FS_MOUNT_DIR/$fsname1 bs=4K   count=1 || rc=1
-	dd if=/dev/zero of=$MOTR_M0T1FS_MOUNT_DIR/$fsname2 bs=128K count=1 || rc=1
-	touch_file $MOTR_M0T1FS_MOUNT_DIR/$fsname3 64 &&
-	dd if=/dev/zero of=$MOTR_M0T1FS_MOUNT_DIR/$fsname3 bs=1M   count=1 || rc=1
+	touch "$MOTR_M0T1FS_MOUNT_DIR"/"$fsname1" || rc=1
```

Comment:
Do you prefer:
```
touch "$MOTR_M0T1FS_MOUNT_DIR/$fsname1" || rc=1
```
instead of 
```
touch "$MOTR_M0T1FS_MOUNT_DIR"/"$fsname1" || rc=1
```
?

It's up to you. I am OK with both of the format.

Reply:
I have used the below in many places and that has been merged also, since this is a cosmetic change, lets go with whatever is already done. 

touch "$MOTR_M0T1FS_MOUNT_DIR"/"$fsname1" || rc=1